src/pybind/mgr/rook/module.py

```diff
@@ -252,7 +252,6 @@ def get_inventory(self, host_filter=None, refresh=False):
 
     @deferred_read
     def get_hosts(self):
-        # type: () -> List[orchestrator.HostSpec]
```

Comment:
revert this?

Reply:
that would overshadow the returntype of the decorator, wouldn't it?

Reply:
This defines the type of the get_hosts without the decorator. The decorator is then applied on top of it.

Reply:
got it, done